CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Version History
     * All Version bumps are required to update this file as well!!
 ----
 
+* 16.0.0 Modifying the Resume model to reflect the changes made in the API (Removal of job title from salary and
+    * addition of id to work experience and work Experience Id to salary
```

Comment:
I don't know what added the extra '*' to the second line, but it doesn't belong. You can see what happens when you add that by using the "rich diff" button in PR view on this file.